inst/help/BainSemBayesian.md

```diff
@@ -0,0 +1,2 @@
+bain SEM
+==========================
```

Comment:
:P

Reply:
Yeah, Herbert will write the Help files once I send him a working version of the Bain SEM analysis :p